### ISSUE TITLE
[BUG] IOS AutOFocus

### ISSUE THREAD

**Issue by sikandernoori** [juliuscanute/qr_code_scanner#431]
Please Fix AutoFocus On IOS, an Android Autofocus is Good and it adjust itself, but on IOS it does not adjust itself rather stays out of focus.

**Comment by jingzhanwu** [comment]
I also encountered the same problem. On IOS, scanning code must keep a certain distance, and focusing is difficult, resulting in insensitive scanning.

**Comment by jnickzlim** [comment]
Yes, encounter this issue as well. IOS Camera app able to focus and scan the qr code, but using qr_code_scanner in app, unable to scan smaller size or blurry qr code.
